addons/website_links/static/src/js/website_links.js

```diff
@@ -533,6 +535,27 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
     },
 });
 
+popupWidget.include({
+    /**
+     * @override
+     */
+    _onHideModal () {
```

Comment:
Maybe worth a triple check in your linter settings, it should highlight this kind of "error":
```suggestion
    _onHideModal() {
```
![image](https://github.com/odoo/odoo/assets/30048408/4711d8af-5d70-4994-863b-e2ca86154007)